Utils/Dataflow/test/utils/reconstruction.py

```diff
@@ -0,0 +1,99 @@
+"""
+Task chain reconstruction from elasticsearch.
+"""
+import copy
+import sys
+
+import elasticsearch
+
+
+INDEX = 'tasks_chain_data'
+
+
+def es_connect():
+    ''' Establish a connection to elasticsearch.
+
+    TODO: take parameters from es config.
+    '''
+    return elasticsearch.Elasticsearch(['localhost:9200'])
+
+
+def get_chain_id(es, index, taskid):
+    ''' Get chain_id by given taskid. '''
```

Comment:
Detailed docstrings are useful not only for automatic documentation production, but also for the cpde readability. I mean the missed parameters and return value descriptions (here and in the other functions) ;)

Reply:
Done. Yes, you are right and I agree with you. It wasn't done before due to the reason I've stated in email - possibility of changes requested by Mikhail that may mean tearing everything down and starting more or less from scratch.